test/TestProjects/MgmtMockAndSample/tests/Generated/Mock/DeletedManagedHsmCollectionMockTests.cs

```diff
@@ -24,17 +24,6 @@ public DeletedManagedHsmCollectionMockTests(bool isAsync) : base(isAsync, Record
             Environment.SetEnvironmentVariable("RESOURCE_MANAGER_URL", $"https://localhost:8443");
         }
 
-        [RecordedTest]
-        public async Task Exists()
```

Comment:
Are these Exists test removed intentionally?

Reply:
Yes. These are convenience methods. They are underlying calling the GET, and we already have mock test generated for GET